eclair-core/eclair-cli

```diff
@@ -26,19 +26,58 @@ where OPTIONS can be:
   -h                    Show this help
   -s                    Some commands can print a trimmed JSON
 
-and COMMAND is one of:
-  getinfo, connect, disconnect, open, close, forceclose, updaterelayfee,
-  peers, channels, channel, allnodes, allchannels, allupdates
-  findroute, findroutetonode, parseinvoice, payinvoice, sendtonode,
-  sendtoroute, getsentinfo, createinvoice, getinvoice, listinvoices,
-  listpendinginvoices, getreceivedinfo, audit, networkfees,
-  channelstats, usablebalances
+and COMMAND is one of the available commands:
```

Comment:
Nice, but this is so easily out-of-date though.

Reply:
I agree, but the previous one was also out-of-date, so while updating it I figured it was easier to keep it up-to-date with that formatting :)
There's probably a lot of improvement we could make to the CLI to make it keep up-to-date more easily, but I don't think that's very important atm.